Header header

geometry_msgs/Quaternion q
float64 wx
float64 wy
float64 wz
float64 ax
float64 ay
float64 az

int32 TargetType  # 1 for Armor, 2 for normal Windwill, 3 for sin Windmill(我知道我风车的英文拼错了，但是我就是不改～～）

int32 Color   #1 for red, 2 for blue